plugins/modules/postgresql_set.py

```diff
@@ -182,7 +182,7 @@
 PG_REQ_VER = 90400
 
 # To allow to set value like 1mb instead of 1MB, etc:
-POSSIBLE_SIZE_UNITS = ("mb", "gb", "tb", "b")
+POSSIBLE_SIZE_UNITS = ("mb", "gb", "tb")
```

Comment:
Is this intended? Why did you remove the `b` in this commit?

Reply:
@tcraxs sorry for late response, I'm in a business trip, I added it in this PR but it's wrong (I'll add an explanation to the comment to this tuple), I'll continue to work on the PR soon, will let you know.

Reply:
I replaced the redundant loop where it's used, could you please take a look